cast/src/receiver/second-load.ts

```diff
@@ -1,3 +1,3 @@
-import "../../../src/resources/ha-style";
-import "../../../src/resources/roboto";
 import "./layout/hc-lovelace";
+
+import("../../../src/resources/ha-style");
```

Comment:
this entire module is already dynamically loaded

Reply:
So just remove the import here?  Was the static import a mistake?

Reply:
I took what I think is a better approach and just removed this module, and instead loaded both in `hc-main`.  That way they are loaded in parallel and the bundle is slightly smaller.